app/views/ReportingFrequencyView.scala.html

```diff
@@ -135,114 +168,152 @@
     }
 }
 
-@dropDownContentOptInOptOutContentUpdateR17 = {
-
-    @p(id = Some("separately-choose-to-opt-out")) {
-        @messages("reporting.frequency.latency.details.section.text.para1.updateR17")
+@latentBusinessText = {
+    @p(id = Some("rf-latency-para1")) {
+        @messages("reporting.frequency.latency.details.section.text.para1")
     }
     <ul class="govuk-list govuk-list--bullet">
-        <li id="latency-section-1-bullet-1">@messages("reporting.frequency.latency.details.section.text.para1.bullet1.updateR17")</li>
-        <li id="latency-section-1-bullet-2">@messages("reporting.frequency.latency.details.section.text.para1.bullet2.updateR17")</li>
+        <li id="rf-latency-para1-bullet1">@messages("reporting.frequency.latency.details.section.text.para1.bullet1")</li>
+        <li id="rf-latency-para1-bullet2">@messages("reporting.frequency.latency.details.section.text.para1.bullet2")</li>
     </ul>
-    @p(id = Some("options-available")) {
-        @messages("reporting.frequency.latency.details.section.text.para2.updateR17")
+    @p(id = Some("rf-latency-para2")) {
+        @messages("reporting.frequency.latency.details.section.text.para2")
     }
     <ul class="govuk-list govuk-list--bullet">
-        <li id="latency-section-2-bullet-1">@messages("reporting.frequency.latency.details.section.text.para2.bullet1.updateR17")</li>
-        <li id="latency-section-2-bullet-2">@messages("reporting.frequency.latency.details.section.text.para2.bullet2.updateR17")</li>
-        <li id="latency-section-2-bullet-3">@messages("reporting.frequency.latency.details.section.text.para2.bullet3.updateR17")</li>
```

Comment:
Same with the dropdown partial/component are we completely replacing this section? I feel we it should be still valid?  

I think it is a new section not replacing the dropdown

<img width="906" alt="Screenshot 2025-07-07 at 17 16 49" src="https://github.com/user-attachments/assets/5e7a5050-e8c6-48f5-b41a-0e944ae2c5d8" />

Reply:
ah okay section is still present but guessing due to git shenanigans the new feature switched content has been removed accidentally. It likely should go back in.